foundation/foundation-mda/src/main/resources/templates/data-delivery-data-records/spark.schema.base.java.vm

```diff
@@ -139,7 +139,8 @@ public abstract class ${record.capitalizedName}SchemaBase extends SparkSchema {
         }
 
         // Remove validation columns from valid data
-        Seq<String> columnsToDrop = JavaConversions.asScalaBuffer(validationColumns).toSeq();
```

Comment:
i: JavaConversions.asScalaBuffer is now deprecated

Reply:
This a candidate for a clean baton migration? If not, we may want to go ahead and leave bread crumbs for upgrading projects in the draft release notes. Something like what package version led to this and a link to their migration docs or release notes.

Reply:
yeah.. I will take in this change.